.rubocop.yml

```diff
@@ -8,6 +8,7 @@ AllCops:
     - 'node_modules/**/*'
     - 'bin/**/*'
     - 'db/schema.rb'
+    - 'vendor/**/*'
```

Comment:
Is this just for your local env?

Reply:
I'm not quite sure where this came from, but it's being linted

Reply:
(and it shouldn't be). It seems to be a gem cache